massa-execution/src/interface_impl.rs

```diff
@@ -51,6 +56,25 @@ impl Interface for InterfaceImpl {
         }
     }
 
+    /// Returns zero as a default if address not found.
+    fn get_balance(&self) -> Result<u64> {
+        let context = context_guard!(self);
+        let address = match context.call_stack.back() {
+            Some(addr) => addr,
+            _ => bail!("Failed to read call stack current address"),
+        };
+        Ok(context.ledger_step.get_balance(&address).to_raw())
```

Comment:
@damip you can set your editor to see the clippy warnings

Reply:
ok